### PR TITLE
Media Services - intros and headings

### PR DESCRIPTION
**Pull request by johnpapa**
added intros for heading sections

needs review before #sign-off

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/media-services.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-12/docs-ref-services/media-services.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180047365251-112%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/media-services.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ ms.service: Media Services
 # Azure Media Services packages for Node.js
```
Comment (clarkio) [inline docs-ref-services/media-services.md]: Change to `Modules`

File: docs-ref-services/media-services.md [hunk 2]
```diff
@@ -24,12 +25,19 @@ With Azure Media Services, you can:
 - Stream your content live or deliver content on-demand. The topic also links to other relevant topics.
 
 ## Management Package
+
 ### Install npm module
+
+Install the Azure media services npm module
```
Comment (clarkio) [inline docs-ref-services/media-services.md]: Capitalize `Azure Media Services`

File: docs-ref-services/media-services.md [hunk 3]
```diff
@@ -24,12 +25,19 @@ With Azure Media Services, you can:
...
+
+Install the Azure media services npm module
+
 ```bash
 npm install azure-arm-mediaservices
 ```
 
 ### Example
+
+This example liss all media services for a resource group
```
Comment (clarkio) [inline docs-ref-services/media-services.md]: Typo `lists`

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/media-services.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-12/docs-ref-services/media-services.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180402468959-112%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/media-services.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-12/docs-ref-services/media-services.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180415583506-112%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@johnpapa @clarkio @jawache merged

**Pull request merged**